modules/code-editor/src/views/full/utils/wrapper.ts

```diff
@@ -14,7 +14,7 @@ const wrapper = {
       return `${HOOK_SIGNATURES[hookType]}{\n${START_COMMENT}\n\n${content}\n${END_COMMENT}\n}`
     } else if (type === 'bot_config') {
       return content.replace('../../bot.config.schema.json', 'bp://types/bot.config.schema.json')
-    } else if (type === 'global_config') {
+    } else if (type === 'main_config') {
```

Comment:
I still don't understand why you renamed it main_config ?

Reply:
since I had bot_config, module_config, (and that module_config could be global or bot), it made more sense to separate them